edubots_surveys.Rnw

```diff
@@ -1440,126 +1438,118 @@ first was to check whether teachers already used some kind of messaging
-need to delegate or offload part
-of the burden of answering, and attending, every single question posed by
-students. In an horizontal setting, other students can answer if the teacher
+The analysis of the questions related to RQ4 seems to indicate that teachers
+prefer to share an interaction space with their students. As
+shown in Tables \ref{tab:kindinteractions} and \ref{tab:interaction}, the majority of responders
+indicated that they {\em do not} want unidirectional communication, and that they prefer
+{\em horizontal} interaction. This point is reinforced by results on the kind of
+chatbots they preferred: most of them indicated FAQ bots, which might indicate a
+need to offload part of the burden of answering every single question posed by
+their students. In a horizontal setting, other students would answer if the teacher
 does not do it immediately. This might also be the reason why the possibility of
 using tools from which analytics can be extracted is also valued by teachers:
-that way, activity in the chat tool, answering questions, regularity, can all be
-observed and valued as part of the process of assessing achievement of learning
-objectives by students.
+in that way, the student activity in the chat tool, answering questions, or their regularity can all be
+assessed as part of the process of achieving learning objectives.
 
 % Only some COVID-response relationship.
-In our survey, we could not find an answer about the right moment for introducing chatbot technologies. Response to questions related to change of
-behavior during and after the first stay at home, stage of the pandemic do not suggest that a major (or minor)
-crisis could be an opportunity to introduce new technologies, since it does not
-bring major changes in attitude (as a minor "crisis" we could mention any institutional event that brings about methodological changes, such as new higher education laws\footnote{We seem to have one of these, at least in
-  Spain, every 10 years or so.}), or simply introduction of new degrees. Although
-changes driven by external forces do offer the chance of piloting new technologies, they do not
-seem to bring internal changes in attitudes in the teaching staff (which, after all, is
+In our survey, we could not find an answer about the right moment to introducing chatbot technologies. The response to questions regarding to change of
+behavior during and after the first stay-at-home stage of the pandemic do not suggest that a major (or minor, such as new higher education laws or the introduction of new degrees) crisis could be an opportunity to introduce new technologies. The reason being that it does not
+bring major changes in attitude 
+%(as a minor "crisis" we could mention any institutional event that brings about methodological changes, such as new higher education laws\footnote{We seem to have one of these, at least in Spain, every 10 years or so.}), 
+%or simply the introduction of new degrees. 
```

Comment:
Why did you delete this?

Reply:
I moved part of it to the brackets in line 1547. It is hard to read, because we mention major and minor crisis and then, when writing about the reason we explain again what a minor crisis is ... 
Sorry, I just realized I left the footnote out, please put it back (in line 1547)

Reply:
Not clear what you mean here. Just modify this PR...